renderdoc/os/posix/linux/linux_stringio.cpp

```diff
@@ -596,7 +596,16 @@ rdcstr GetTempRootPath()
 rdcstr GetAppFolderFilename(const rdcstr &filename)
 {
   passwd *pw = getpwuid(getuid());
-  const char *homedir = pw->pw_dir;
+  const char *homedir = pw ? pw->pw_dir : nullptr;
```

Comment:
Minor nit I didn't see before, please use `NULL` instead of `nullptr`.

Reply:
Done.